paapi.go

```diff
@@ -238,20 +242,34 @@ func (client *PAExternalAPIClient) GetResources(groupFilters []GroupFilter) (*Mo
 				if err != nil {
 					return nil, err
 				}
+				metricTitles := make(map[string]int)
 				for _, metric := range values.Infos {
-					newMetric := MonitoredValue{
-						GroupID:        group.ID,
-						ServerID:       server.ID,
-						MonitorValue:   metric.Status,
-						MonitorStatus:  metric.Status,
-						MonitorTitle:   metric.Title,
-						MonitorLastRun: metric.LastRun.Time,
+					// metric title is not unique
+					// discarding duplicate monitors
+					if _, titleExists := metricTitles[metric.Title]; titleExists {
+						metricTitles[metric.Title]++
+						log.Warnf("Duplicate monitor %s for server %s!", metric.Title, server.Name)
```

Comment:
As duplicate is by group also, I would add this precision:
```suggestion
						log.Warnf("Duplicate monitor %s for server %s and group path %s. This monitor will be ignored.", metric.Title, server.Name, group.Path)
```